### ISSUE TITLE
Pet sometimes doesn't load after restart

### ISSUE THREAD

**Issue by Kyzderp** [Kyzderp/ArmorStandPetWiki#1]
Occurs only after restart, and logging in at the exact location it was. Teleporting away and back (to reload the chunks) fixes this.

**Comment by JohnCool12** [comment]
Sorry for asking here, I don't know anywhere else to contact you. Do you have the link for the ArmorStandPet Spigot plugin? It is no longer present on the Spigot website.

**Comment by Kyzderp** [comment]
> Sorry for asking here, I don't know anywhere else to contact you. Do you have the link for the ArmorStandPet Spigot plugin? It is no longer present on the Spigot website.

Heya, I certainly wasn't expecting any activity here xD I haven't touched plugin dev in over 6 years, I guess Spigot removes for inactivity or something? I opened the source here https://gitlab.com/Kyzderp/ArmorStandPet so you could build it yourself if you want to. If you need the jars, I do have the old files but don't know what's in each version, and they're of course for very old Minecraft versions. I don't remember how to build or even use Eclipse anymore at this point :D

**Comment by JohnCool12** [comment]
Yes I would love the old jar files please. To be honest I am perfectly fine playing on old versions of Minecraft. The plugin is just that good.

**Comment by Kyzderp** [comment]
I think I've added them as a release here https://github.com/Kyzderp/ArmorStandPetWiki/releases/tag/Legacy

Please let me know if that doesn't work!

**Comment by JohnCool12** [comment]
The releases page works! Thank you for the .jar files!! :D
